### PR TITLE
Forte-RC ROS package added

### PR DESCRIPTION
**Pull request by MicaelCouceiro**

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -2297,6 +2297,23 @@ repositories:
       url: https://github.com/kth-ros-pkg/force_torque_tools.git
       version: indigo
     status: maintained
+  forte_rc:
+    doc:
+      type: git
+      url: https://github.com/ingeniarius-ltd/forte_rc.git
+      version: indigo
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: The repository does not have an `indigo` branch.

**Comment by vrabaud** [comment]
@MicaelCouceiro , can you please create an ```indigo``` branch. Can you also make sure your release repo is a proper one.
Was that PR a test ? Should we close it ?

**Comment by MicaelCouceiro** [comment]
Dear Vincent,

 

Thank you for your concern. We are still finishing some changes on the code and we will soon upload it to github. Is that ok for you or you prefer to close it for now?

 

Kind regards,

 


Micael Couceiro

Ingeniarius, CEO

 <mailto:micael@ingeniarius.pt> micael@ingeniarius.pt

	

Ingeniarius, Lda.


Rua da Vacariça, nº 37  
3050-381 MEALHADA

Tel :  + 351 96 117 14 09

 <http://www.ingeniarius.pt/> www.ingeniarius.pt

 <http://www.ingeniarius.pt/> 

Find us on

 

 <https://www.facebook.com/ingeniarius.pt>   <http://www.linkedin.com/company/ingeniarius>

File: indigo/distribution.yaml [hunk 2]
```diff
@@ -2297,6 +2297,23 @@ repositories:
...
+    release:
+      packages:
+      - FORTERC_driver
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/ingeniarius-ltd/forte_rc-release.git
```
Comment (tfoote) [inline indigo/distribution.yaml]: This repository is empty too.

**Comment by tfoote** [comment]
Both the source repository and release repository you've asked us to index are empty. I'm going to close these as we cannot index your repositories until they have content. It will both give information that is incorrect to users and cause our buildfarm to keep retrying and failing to process your repositories.

Also it appears that you manually generated this pull-request. It's highly recommended to let bloom do it for you as it will embed the right information. See the [bloom tutorials](http://wiki.ros.org/bloom/Tutorialshttp://wiki.ros.org/bloom/Tutorials)

**Pull request closed without merging**